app/controllers/courses_controller.rb

```diff
@@ -28,6 +28,21 @@ def self.registered(app)
 
             @course_coll = app.settings.courses_db.collection("courses#{params[:semester]}")
             @section_coll = app.settings.courses_db.collection("sections#{params[:semester]}")
+            @prof_coll = app.settings.courses_db.collection("profs#{params[:semester]}")
+          end
+
+
+          # Route for professors, use 'name' or 'course' to filter
+          app.get '/v0/courses/professors' do
```

Comment:
I've been looking around the UMD websites and it looks like the only real way to get a full list of professors is to either go through every individual department's faculty list or to get them through the sections. It doesn't look like there's a good way to access professors outside of their courses.